components/ws-manager-bridge/src/config.ts

```diff
@@ -33,6 +33,7 @@ export interface Configuration {
     timeouts: {
         metaInstanceCheckIntervalSeconds: number;
         preparingPhaseSeconds: number;
+        buildingPhaseSeconds: number;
```

Comment:
how do we distribute config changes? do we need to have default values for compatibility?
what's the plan for self-hosted?

Reply:
@andrew-farries, I believe this needs to be reflected in configmap.go, like the other timeouts.

Reply:
Done in a0e4ea775123ea0476c2c43814f189f6074d183e